packages/appium/docs/en/contributing.md

```diff
@@ -0,0 +1,123 @@
+---
+hide:
+  - navigation
+
+title: Contributing to Appium
+---
+
+The Appium project would not exist without the many contributions of code, documentation,
+maintenance, and support from companies and volunteers. As such, we welcome contributions!
+
+There are a lot of different ways to help the project - see below for everything you can do and the
+processes to follow for each contribution method. Note that no matter how you contribute, your
```

Comment:
missing comma after `Note`

Reply:
I believe `that` is used as a substitute for a comma

Reply:
`Note that` is correct